packages/manager/modules/billing/src/components/split-payment/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,16 @@
+{
+  "billing_payment_method_split_payment_title": "Paiement fractionné",
+  "billing_payment_method_split_payment_information": "Le mécanisme de paiement fractionné s'applique aux sociétés, associations et entités liées ou contrôlées par l'administration publique qui figurent sur les listes officielles et qui peuvent être consultées ci-dessous.",
+  "billing_payment_method_split_payment_activate": "Activer le paiement fractionné",
+  "billing_payment_method_split_payment_deactivate": "Désactiver le paiement fractionné",
+  "billing_payment_method_split_payment_cancel": "Annuler",
+  "billing_payment_method_split_payment_active": "Actif",
+  "billing_payment_method_split_payment_inactive": "Inactif",
+  "billing_payment_method_split_payment_activating": "En attente d'approbation",
+  "billing_payment_method_split_payment_deactivating": "En cours de désactivation",
+  "billing_payment_method_split_payment_information_link": "Information sur le paiement fracturé",
+  "billing_payment_method_split_payment_activate_success": "Votre demande d'activation du paiement fractionné a bien été prise en compte.",
+  "billing_payment_method_split_payment_activate_error": "Une erreur est survenue lors de votre demande d'activation du paiement fractionné.",
+  "billing_payment_method_split_payment_deactivate_success": "Votre demande de désactivation du paiement fractionné a bien été prise en compte.",
+  "billing_payment_method_split_payment_deactivate_error": "Une erreur est survenue lors de votre demande de désactivation du paiement fractionné."
```

Comment:
Not displaying the error message ?

Reply:
Error message is appended to this sentence, no need to have it in here as a parameter of the translation